web/src/components/lineage/Lineage.tsx

```diff
@@ -23,6 +23,8 @@ import MqEmpty from '../core/empty/MqEmpty'
 import MqText from '../core/text/MqText'
 import Node from './components/node/Node'
 import ParentSize from '@visx/responsive/lib/components/ParentSize'
+import { useTranslation } from 'react-i18next'
+import i18next from '../../i18n'
```

Comment:
@merobi-hub I think you don't want to be importing this here. This should be your type issue for `no-implicit-any`